packages/react/src/components/output/Output.tsx

```diff
@@ -85,29 +98,18 @@ export const Output = (props: IOutputProps) => {
                 const out = val.data['text/html']; // val.data['application/vnd.jupyter.stdout'];
                 if (out) {
                   if ((out as string).indexOf(receipt) > -1) {
-                    outputStore.setGrade({
-                      sourceId,
-                      success: true,
-                    });
+                    outputStore.setGradeSuccess(sourceId, true)
                   }
                 }
               }
             });
           }
         });
       }
-      setAdapter(adapter);
-      outputStore.setAdapter(sourceId, adapter);
-      adapter.outputArea.model.changed.connect((outputModel, args) => {
-        setOutputs(outputModel.toJSON());
-      });
     }
   }, [id, kernel]);
```

Comment:
Connected signal should be disconnected as part of the `useEffect` clean-up returned callback

Reply:
Thx for @fcollonval for the review and help an this. Fixed with https://github.com/datalayer/jupyter-ui/pull/282/commits/771d78096376ba5f4d007ccc8e4bed81801236ff